test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <PublishAot Condition="!$([MSBuild]::IsOSPlatform('OSX'))">true</PublishAot>
```

Comment:
You could also remove this - OSX is now supported for NativeAOT as well

Reply:
removed; thank you!